<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>967</width>
    <height>787</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>461</width>
      <height>721</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Function TYpe</string>
    </property>
    <widget class="QRadioButton" name="radioButton_SVFunction">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>f(x)</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonTVFunction">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>z(x, y)</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>441</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Generate Table</string>
    </property>
    <widget class="QLabel" name="labelNPoints">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>N points:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxNPoints">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>44</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxSameWeight">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Same weight</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxSameWeight">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>91</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonGenerateTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>441</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSetAllWeightsSame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>441</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Set all weights the same</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>967</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
